src/test/java/UploadReceiptServletTest.java

```diff
@@ -93,6 +93,8 @@
   private static final String LABEL = "Label";
   private static final Text RAW_TEXT = new Text("raw text");
   private static final double PRICE = 5.89;
+  private static final double PRICE_THREE_DECIMAL_PLACES = 17.236;
+  private static final double PRICE_THREE_DECIMAL_PLACES_ROUNDED = 17.24;
```

Comment:
If the variable isn't going to be used by more than one test, I prefer to place it inside the test method. This way the list of class variables doesn't become long and unwieldy.